src/util/colors.ts

```diff
@@ -1,3 +1,4 @@
+export const LIGHT_BLUE = '#768FC0';
```

Comment:
Do we need Error Red? #FF0000

Reply:
You're right! our current red is not up to date. i'll update the red color within #44